knowledge-base/grid-filterrow-bool-customization.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Change the Boolean Filtering Options in a FilterRow Dropdown
+description: Learn how to customize the default text in a Grid FilterRow dropdown list, including changing "is true" and "is false" to "Yes" and "No," modifying the FilterRow dropdown list display names, and filtering the boolean column values by null for a more user-friendly and tailored filtering experience.
+type: how-to
+page_title: How to Change the Boolean Filtering Options in a FilterRow Dropdown
+slug: grid-kb-filterrow-bool-customization
+ticketid: 1575619, 1603347, 1646267, 1654640
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to change the default `is true` and `is false` text in the Grid the `FilterRow` dropdown?
+* How to change the default display names in `FilterRow` dropdown?
+* How to filter boolean `Grid` column values by `null`?
+* How to change the default values in `FilterRow` dropdown to `Yes` and `No`?
```

Comment:
Do you have any considerations that make you avoid the article before `FilterRow`?
```suggestion
* How to change the default values in the `FilterRow` dropdown to `Yes` and `No`?
```